src/main/java/com/example/nlern/repos/TestRepository.java

```diff
@@ -6,6 +6,7 @@
 import java.util.List;
 
 public interface TestRepository extends CrudRepository<Test, Integer> {
-    List<Test> findTop5ByAuthorIdOrderByIdDesc(int Author);//findTop5ByOrderByIdDesc();
+    List<Test> findTop5ByAuthorIdOrderByIdDesc(int Author);
```

Comment:
Same here 
int Author, isn't Author a whole class?

Reply:
Yes, it is. Corrected